flink-end-to-end-tests/test-scripts/test_kubernetes_embedded_job.sh

```diff
@@ -18,12 +18,7 @@
 ################################################################################
 
 source "$(dirname "$0")"/common.sh
```

Comment:
nit: we can remove this because it is already covered in `common_kubernetes.sh`?

Reply:
Yes, i will remove this line.